wiggum_app/models.py

```diff
@@ -127,6 +137,27 @@ def getDistanceHeatmapDict(df):
 
     return distance_heatmap_dict_list
 
+def addTrendDisplayName(df):
+    """
+    Add trend display name column to df.
+
+    Parameters
+    -----------
+    df : DataFrame
+        dataframe
```

Comment:
note here that it has to be a result_df

Reply:
It could be result_df, filter_df, rank_df and detect_df, therefore, I use a general name df here.

Reply:
it can't be a data_df. and rilter, rank and detect results are all result_dfs (in terms of the column names, structure, etc)

Reply:
for example, it has to have a trend_type column

Reply:
For this, I also change it to cur_result_df. Is it ok?

Reply:
Changed